modules/proxyconnection/ProxyConnectionPC.js

```diff
@@ -80,26 +80,26 @@ export default class ProxyConnectionPC {
     /**
      * Updates the peer connection based on the passed in jingle.
      *
-     * @param {Object} $jingle - An XML jingle element, wrapped in query,
+     * @param {Object} jingle - An XML jingle element, wrapped in query,
```

Comment:
Please update the jsDoc, it's no longer wrapped in jQuery.

Reply:
Will do